priv/repo/migrations/20180702172407_add_log_category.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Spender.Repo.Migrations.AddLogCategory do
+  use Ecto.Migration
+  @moduledoc """
+  This module holds the database schema for the logCategory table
+  we use this table to records the logExpense when we log in an expense
+  """
+  @doc """
```

Comment:
Could you consider spacing these better perhaps adding a blank line between these two comments